docs/content/configuration/global-configuration/configmap-resource.md

```diff
@@ -185,6 +185,7 @@ See the doc about [VirtualServer and VirtualServerRoute resources](/nginx-ingres
 |``app-protect-failure-mode-action`` | Sets the ``app_protect_failure_mode_action`` [global directive](/nginx-app-protect/configuration/#global-directives). | ``pass`` |  |
 |``app-protect-cpu-thresholds`` | Sets the ``app_protect_cpu_thresholds`` [global directive](/nginx-app-protect/configuration/#global-directives). | ``high=100 low=100`` |  |
 |``app-protect-physical-memory-util-thresholds`` | Sets the ``app_protect_physical_memory_util_thresholds`` [global directive](/nginx-app-protect/configuration/#global-directives). | ``high=100 low=100`` |  |
+|``app-protect-reconnect-period-seconds`` | Sets the ``app_protect_reconnect_period_seconds`` [global directive](/nginx-app-protect/configuration/#global-directives). Value type: number with decimal fraction. Value Range: 0-60. 0 is illegal. | ``5`` |  |
```

Comment:
I think this wording is strange, and may point to an underlying change that can make it more natural.
- "Decimal fraction" is not commonly used.
- "Range 0-60. 0 is illegal" is strange. Ranges are generally inclusive, or alternatively include the first number but exclude the second. I've never seen a range that excluded the first number.

Generally when time measurements require precision, I've seen milliseconds used, not seconds. This is the equivalent of seconds with three decimal places. It also makes the range description more readable, without the need for '0 is illegal'. We instead say "Range: 1-60000"

Can we either
a) Change unit to milliseconds to keep precision and eliminate awkward wording, or
b) Keep as seconds but remove the precision (value always rounded to the next whole second)

Reply:
Agreed, this is unclear as written. However, as this is verbatim how the directive is described in the App Protect docs, this PR may not be the appropriate forum for requesting changes. It's sufficient to link to the directive in the NAP WAF docs here instead of duplicating the information.

```suggestion
|`app-protect-reconnect-period-seconds` | Sets the `app_protect_reconnect_period_seconds` [global directive](/nginx-app-protect/configuration/#global-directives). | `5` |  |
```

Reply:
Thanks @jputrino , that's a good way around it.